src/test/java/de/uni_passau/fim/se2/pipeline_helper/checkers/FileExistsCheckerTest.java

```diff
@@ -65,12 +68,40 @@ void shouldRecogniseNonExistingFiles() throws Exception {
     }
 
     @Test
-    void shouldRecogniseEmptyAndNonExistingFiles() throws Exception {
+    void shouldRecogniseNonReadableFiles() throws Exception {
+        final FileExistsChecker checker = new FileExistsChecker(
+            List.of(
+                resource("line_length_checker_demo_files/illegal_byte_sequence/InvalidByteSequence.java")
+            )
+        );
+
+        final CheckerResult result = checker.check();
+        assertThat(result.isSuccessful()).isFalse();
+        assertThat(result.getMessage()).isEqualTo("""
+            Non-readable files:
+            InvalidByteSequence.java""");
+    }
+
+    @Test
+    void shouldNotConsiderAnEmptyDirectoryAFile() throws Exception {
```

Comment:
```suggestion
    void shouldNotConsiderADirectory() throws Exception {
```
The new `Tests.txt` directory is not empty. (I think you could have used any of the pre-existing directories for the test instead.)

Reply:
You're right 🤦‍♂️ Will do that then